README.md

```diff
@@ -1,27 +1,30 @@
-**Reatom is a ultimate logic and state manager for small widgets and huge SPAs.**
+**Reatom is the ultimate logic and state manager for small widgets and huge SPAs.**
 
 ## Key features
 
 - **simple** and powerful abstractions.
-  <small>There are only three main primitives: `ctx`, `atom`, `action`, all other features and packages works on top of that.</small>
+  <small>There are only three main primitives: `ctx`, `atom`, `action`. All other features and packages work on top of that.</small>
 - **immutable** and reliable.
-  <small>All pure computations processed with atomicity guaranties.</small>
+  <small>All pure computations processed with atomicity guarantees.</small>
 - **explicit reactivity** without proxies.
-  <small>To archive [maximum](#how-performant-reatom-is) performance we have [atomization](https://www.reatom.dev/recipes/atomization) pattern.</small>
+  <small>We use the [atomization](/recipes/atomization) pattern to achieve [maximum](#how-performant-reatom-is) performance </small>
 - perfect **effects management**.
-  <small>Advanced [async package](https://www.reatom.dev/package/async) allows you to describe complex async flows with caching, retrying and automatic cancellation with native `await` and `AbortController`.</small>
+  <small>Advanced [async package](/package/async) allows you to describe complex async flows, including caching, retrying and automatic cancellation with native `await` and `AbortController`.</small>
 - nice **debugging** experience.
-  <small>Each atom and action update stores [immutable cause (call) stack](https://www.reatom.dev/getting-started/debugging/) which is super helpful for debugging complex async flows. To simplify this we have [logger package](https://www.reatom.dev/package/logger).</small>
+  <small>Each atom and action updates the store's [immutable cause (call) stack](/getting-started/debugging/). It helps a&nbsp;lot in debugging complex async flows. We also provide a [logger package](/package/logger) for that.</small>
```

Comment:
Should we use `ctx` instead of `store` here? 🤔 
`store` might be confusing because there is no `store` in Reatom

```suggestion
  <small>Each atom and action updates the ctx's [immutable cause (call) stack](/getting-started/debugging/). It helps a&nbsp;lot in debugging complex async flows. We also provide a [logger package](/package/logger) for that.</small>
```

Reply:
Store is a legacy term used in v2. You can change it to Context